pkg/json/decode.go

```diff
@@ -20,7 +22,39 @@ func UnmarshalBody(body io.Reader, req interface{}) error {
 		return err
 	}
 
-	err = getValidator().Struct(req)
+	return validateStruct(req)
+}
+
+func UnmarshalJSON(src, dest interface{}) error {
+	bdata, err := MarshalJSON(src)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(bdata, dest)
+	if err != nil {
+		return err
+	}
+
+	return validateStruct(dest)
+}
+
+func UnmarshalYAML(src, dest interface{}) error {
```

Comment:
We sre under pkg json, this utility does not seem to belong here

Reply:
this is not possible currently because the `json` and `validate` have been merged together inside of the `json` package. I created a ticket to fix this in the futrure and remove the `validate` from the `json`: https://app.zenhub.com/workspaces/tanuki-5ea70772b186e10067f57842/issues/consensys/quorum-key-manager/430